e2e/less/yarn.lock

```diff
@@ -0,0 +1,408 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@bazel/less@file:../../dist/bin/packages/less/npm_package":
+  version "1.2.3"
+  dependencies:
+    less "^3.9.0"
+
+ajv@^6.5.5:
+  version "6.10.2"
+  resolved "https://nexus-binrepo.sapphirepri.com/nexus/content/groups/npm-all/ajv/-/ajv-6.10.2.tgz#d3cea04d6b017b2894ad69040fec8b623eb4bd52"
```

Comment:
oops, this file needs to be regenerated without your custom registry mirror - this fails on CI

Reply:
I knew this was going too smoothly. Nice catch!